src/js/modules/infragistics.ui.validator.js

```diff
@@ -2257,58 +2257,61 @@ $.ig.igValidatorValueRule = $.ig.igValidatorValueRule || $.ig.igValidatorNumberR
 		return this._lastMessageType;
 	},
 	isValid: function(options, value) {
-		var min, max,
+		var min = false, max = false, //error flags
+			minValue, maxValue, hasMin, hasMax,
 			isNumber = this._isNumber(options, value),
 			isDateParsable = !isNaN(new Date(value).getSeconds());
 
-		if (isDateParsable || isNumber) {
-			var minValue = options.valueRange.push ? options.valueRange[ 0 ] : options.valueRange.min,
-				maxValue = options.valueRange.push ? options.valueRange[ 1 ] : options.valueRange.max,
-
-				// must be type checked, 0 should be valid
-				hasMin = typeof minValue === "number" || minValue,
-				hasMax = typeof maxValue === "number" || maxValue;
-
-			if ((hasMin || hasMax)) {
-				if (isNumber && !options.date) {
-					value = this._parseNumber(value, options);
-					min = hasMin && minValue;
-					min = value < min ? min.toString() : null;
-					max = hasMax && maxValue;
-					max = value > max ? max.toString() : null;
-				} else if (isDateParsable && !options.number) {
-					value = new Date(value);
-					if (hasMin) {
-						min = minValue = new Date(minValue);
-						minValue = minValue.toLocaleString();
-					}
-					min = value < min ? min.toLocaleString() : null;
-
-					if (hasMax) {
-						max = maxValue = new Date(maxValue);
-						maxValue = maxValue.toLocaleString();
-					}
-					max = value > max ? max.toLocaleString() : null;
-				}
+		if (!isDateParsable && !isNumber) {
+			//can't be handled by this rule
+			return true;
+		}
+		minValue = options.valueRange.push ? options.valueRange[ 0 ] : options.valueRange.min;
+		maxValue = options.valueRange.push ? options.valueRange[ 1 ] : options.valueRange.max;
```

Comment:
Actually it won't - it'll be undefined just as it would be if the option is set with just `{ min: 3}` object. There are tests running with just one item in the array too.